code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -450,3 +450,7 @@
 
 /obj/structure/closet/AllowDrop()
 	return TRUE
+
+
+/obj/structure/closet/return_temperature()
+	return
```

Comment:
Why does this need to exist?

Reply:
there was a bug where you'd be temperature proof inside lockers IIRC